app/src/main/res/layout/image_card_view.xml

```diff
@@ -38,7 +38,8 @@
             android:layout_height="25dp"
             android:src="@drawable/ic_heart_red"
             android:alpha="0.8"
-            android:layout_alignBottom="@+id/main_image"
+            android:layout_alignBottom="@+id/resumeProgress"
+            android:layout_marginBottom="2dp"
```

Comment:
I'd say the margin should be 5dp on the left,bottom and right but simply using:
```
            android:layout_marginStart="5dp"
            android:layout_marginEnd="5dp"
            android:layout_marginBottom="5dp"
```

Will make the progressbar too wide, probably another property should be changed for that too.

Reply:
![image](https://user-images.githubusercontent.com/2305178/99121816-65cfc180-25fd-11eb-8110-eef55e5e9bc7.png)

(don't mind the weird scaling, I screwed up my emulator settings)

Reply:
What weird scaling? It may actually be an issue you just think is scaling lol. If you are wanting to make the progress bar 5dp from the bottom and sides, the padding is what is used for the left and right.

Reply:
The scaling issues was definitely me using "adb wm" to change some things :p. They're not really noticeable in this screenshot though.